frontend/javascript/components/postupload/post-upload.vue

```diff
@@ -0,0 +1,1685 @@
+<script setup>
+import { ref, reactive, computed, defineProps, nextTick } from 'vue'
+import SimpleStepper from './simple-stepper.vue'
+// import PublicbodyChooser from '../publicbody/publicbody-chooser'
+import PublicbodyChooser from '../publicbody/publicbody-beta-chooser'
+// TODO linter wrong? the two above are just fine...
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import DocumentUploader from '../docupload/document-uploader.vue'
```

Comment:
I guess this is because in line 190, you define `const documentUploader = ref()`. This introduces ambiguity, since component names are case-insensitive. While Vue uses the correct one (the imported component) in rendering, it seems ESLint doesn't like it so much.